src/app/app.routes.ts

```diff
@@ -1,3 +1,10 @@
 import { Routes } from '@angular/router';
+import { TodoAddingComponent } from './todo/components/todoAdding/todoAdding.component';
+import { TodoListComponent } from './todo/components/todoList/todoList.component';
+import { TodoItemComponent } from './todo/components/todoItem/todoItem.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  { path: 'todoAdd-component', component: TodoAddingComponent },
```

Comment:
Please try to use angular syntax for component naming, the default is app-etcetc but if you want you can use your own prefix, but it should be used for all components

Reply:
also to mention the component names are divided by - so todo-add instead of todoAdd